src/main/java/uk/gov/mca/beacons/api/jpa/entities/Person.java

```diff
@@ -9,13 +9,19 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.validation.constraints.Email;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 import uk.gov.mca.beacons.api.domain.PersonType;
 
 @Entity(name = "person")
 @Getter
 @Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
I'm not sure how I feel about annotating these on here.  I personally prefer leaving the JPA entities as plain as possible rather than having builders/all args/no arg constructors.

The JPA spec requires you to [always have a no-arg constructor](https://openjpa.apache.org/builds/1.2.3/apache-openjpa/docs/jpa_overview_pc.html#jpa_overview_pc_no_arg) which is implied in Java if you don't define a constructor and it can be tricky to work out why things have failed to persist with adding stuff like builders/different constructors.

I'm not massively fussed, but if it's just used within the test setup I would be tempted to say leave these off and just create the entity through the no arg constructor and use the setter on the class.